packages/web-components/src/components/rux-pop-up/rux-pop-up.tsx

```diff
@@ -55,6 +55,11 @@ export class RuxPopUp {
      */
     @Prop() placement: ExtendedPlacement = 'auto'
 
+    /**
+     * Turns autoUpdate on or off which makes the pop-up move to stay in view based on scroll. Defaults to false.
+     */
+    @Prop({ mutable: true, reflect: true }) disableAutoUpdate: boolean = false
```

Comment:
we don't need to mark this as mutable since it doesnt change. I also dont think we need to reflect it but I dont feel strongly. Reflecting is absolutely necessary if we're doing :host[attr] styles but thats not too relevant here .

Reply:
ok all set!